public/app/features/provisioning/FileStatusPage.tsx

```diff
@@ -75,19 +82,23 @@ function ResourceView({ wrap, repo, repoRef, tab }: Props) {
   const existingName = wrap.resource?.existing?.metadata?.name;
   const location = useLocation();
   const [queryParams] = useQueryParams();
+  const [replaceFile, replaceFileStatus] = useReplaceRepositoryFilesWithPathMutation();
+
+  const [jsonView, setJsonView] = useState('');
 
-  let showJSON = '';
-  switch (tab) {
-    case TabSelection.Existing:
-      showJSON = JSON.stringify(wrap.resource.existing, null, '  ');
-      break;
-    case TabSelection.DryRun:
-      showJSON = JSON.stringify(wrap.resource.dryRun, null, '  ');
-      break;
-    case TabSelection.File:
-      showJSON = JSON.stringify(wrap.resource.file, null, '  ');
-      break;
-  }
+  useEffect(() => {
+    switch (tab) {
+      case TabSelection.Existing:
+        setJsonView(JSON.stringify(wrap.resource.existing, null, '  '));
```

Comment:
```suggestion
        setJsonView(JSON.stringify(wrap.resource.existing, null, 4));
```
You can also use numbers for the number of space characters.

Reply:
i had no ideaa!